### DISCUSSION TITLE
What's the best way to import to Figma?

### DISCUSSION THREAD

**Discussion by nareshbhatia** [lucide-icons/lucide#1644] [Q&A]
My team is embarking on a new design system project. I suggested our designers to use Lucide Icons because:

1. They are beautiful and consistent
2. There is a large open source community behind it
3. Engineers can easily import them into React

The designers use Figma and they would like to use a popular closed source library that has first class support for Figma, with all icons nicely categorized and componetized. I know there are several files on Figma community that provide Lucide Icons, however it is unclear how well they are structured/supported. Designers are looking for officially supported design files.

**Question**: Is there an easy way to import Lucide icons into a Figma library with the following requirements from our design team:

1. Icons should be flattened (1 stroke layer)
2. Icons should be Figma components
3. Icons should be cleanly tagged and categorized

**Comment by zoharma** [comment]
@nareshbhatia There is a [Figma plugin](https://www.figma.com/community/plugin/939567362549682242) which allows you to build your own icon library, and recently I found in the discussion board here [this great template](https://www.figma.com/community/file/1204720733890022011) which you can publish as a library, and also allows you to change the weight globally through variables.
